R/collect.R

```diff
@@ -64,10 +64,10 @@
 #' However, this means that if a function was supplied to a
 #' [control function's][control_grid()] `extract` argument, tune can only
 #' execute that extraction on the one model that was fitted. As a result,
-#' in the `collect_extracts()` output, tune opts to associate the 
-#' extracted objects with the hyperparameter combination used to 
-#' fit that one model workflow, rather than the hyperparameter 
-#' combination of a submodel. In the output, this appears like 
```

Comment:
Are there `.Rd` changes associated with these deletions?

Reply:
no :( i couldn't figure out how to get rid of it

Reply:
all good!